extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -12,45 +12,118 @@ function __construct( ) {
 	}
 
 	public function index() {
```

Comment:
Made a few updates in this function to reduce indentation and also make sure to return True/False from the method since returning false is necessary to inhibit the template from generating when there is no content or an error.